packages/cashier/src/containers/cashier/cashier.tsx

```diff
@@ -203,38 +198,6 @@ const Cashier = observer(({ history, location, routes: routes_config }: TCashier
         }
     }, [history, is_onramp_visible]);
 
-    useEffect(() => {
-        if (is_p2p_enabled_success && !is_p2p_enabled && history.location.pathname.startsWith(routes.cashier_p2p)) {
-            const url_params = new URLSearchParams(history.location.search);
-            const advert_id = url_params.get('advert_id');
-
-            history.push(routes.cashier_deposit);
-
-            if (advert_id) {
-                if (is_virtual) {
-                    toggleReadyToDepositModal();
-                } else {
-                    error.setErrorMessage({
-                        code: 'ShareMyAdsError',
-                        message:
-                            currency !== 'USD' && is_svg
-                                ? localize('Deriv P2P is currently unavailable in this currency.')
-                                : localize('Deriv P2P is currently unavailable in your country.'),
-                    });
-                }
-            }
-        }
-    }, [
-        currency,
-        error,
-        history,
-        is_p2p_enabled,
-        is_p2p_enabled_success,
-        is_svg,
-        is_virtual,
-        toggleReadyToDepositModal,
-    ]);
-
```

Comment:
this is all related to share my ads edge cases, so this can be removed. here is the figma file: https://www.figma.com/design/4MOUx8arWrBnUJvA3LufAh/branch/bTQkniOz9NeT3fXVFXs1Ns/P2P---My-ads?node-id=21750-322071&node-type=instance&t=g2DYm5qkh9mhZwf7-0

<img width="969" alt="Screenshot 2024-12-11 at 11 31 07 AM" src="https://github.com/user-attachments/assets/bca2f0b2-589b-40e4-be1e-6ff5a6b73104">
<img width="940" alt="Screenshot 2024-12-11 at 11 31 33 AM" src="https://github.com/user-attachments/assets/a9825aae-9395-443a-81f7-667394885747">
<img width="927" alt="Screenshot 2024-12-11 at 11 31 37 AM" src="https://github.com/user-attachments/assets/731b9d6a-2e65-4a60-bbab-6eb617d29a08">